### DISCUSSION TITLE
Issues with Reacts-Markdown in Android 8.0

### DISCUSSION THREAD

**Discussion by liukkm** [remarkjs/remark#1358] [Q&A]
I'm working on an Android tablet compatibility job. The nextjs+react-markdown I use works fine on a computer or later version of Android, but now it keeps coming up on the current version of Android 8.0: `SyntaxError: Unexpected token.`, and I feel overwhelmed. Later, I lowered the react-markdown version from 8.0.7 to 5.0.3 and did not report the corresponding error message, but the plug-in react-math, which used to parse mathematical formulas, started reporting the same error. Later, after I reduced the version of remark-math to the version that supports cjs, no error was reported but the corresponding mathematical formula could not be parsed. Then remark-katex began to report the same error. I'm almost sure it's because of the esm conversion to cjs, but I can't solve these problems, asking if react-markdown and the corresponding rendering plugin have a higher version compatible with Android 8 and CJS

**Comment by remcohaszing** [comment]
If you need to support older browsers, you should use something like Babel to compile newer syntax into something old browsers understand.

**Reply by liukkm** [reply]
Yes, babel has been configured in my project, and webpack has also been configured, zustand idb and so on can take effect according to babel, but react-markdown and the corresponding plug-in have not taken effect.

**Reply by wooorm** [reply]
Look into Babels docs, change your configuration.
